tests/tests/image_atomics/mod.rs

```diff
@@ -5,6 +5,35 @@ use wgpu_test::{
     fail, gpu_test, image::ReadbackBuffers, GpuTestConfiguration, TestParameters, TestingContext,
 };
 
+#[gpu_test]
+static IMAGE_64_ATOMICS: GpuTestConfiguration = GpuTestConfiguration::new()
```

Comment:
Just double checking, we've tested this on all platforms right?

Reply:
it works on mac windows and linux, haven't tried anything else